src/content/docs/2/guide/frontend/nuxt.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Nuxt
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+Nuxt is a meta-framework for Vue.
+
+## Checklist
+
+- Use SSG by setting `ssr: false`. Tauri doesn't support server based solutions.
+- Set host to `0.0.0.0`.
+- Use `dist/` as `distDir` in `tauri.conf.json`.
+- Compile using `nuxi generate`.
+
+## Example Configuration
+
+<Tabs>
+	<TabItem label="npm">
+	
+```json
+// tauri.conf.json
+{
+	"build": {
+		"beforeDevCommand": "npm run dev",
+		"beforeBuildCommand": "npm run generate",
+		"devPath": "http://localhost:3000",
+		"distDir": "../dist"
+	}
+}
+```
+	
+	</TabItem>
+	<TabItem label="yarn">
+	
+```json
+// tauri.conf.json
+{
+	"build": {
+		"beforeDevCommand": "yarn dev",
+		"beforeBuildCommand": "yarn generate",
+		"devPath": "http://localhost:3000",
+		"distDir": "../dist"
+	}
+}
+```
+	
+	</TabItem>
+	<TabItem label="pnpm">
+	
+```json
+// tauri.conf.json
+{
+	"build": {
+		"beforeDevCommand": "pnpm dev",
+		"beforeBuildCommand": "pnpm generate",
+		"devPath": "http://localhost:3000",
+		"distDir": "../dist"
+	}
+}
+```
+	
+	</TabItem>
+</Tabs>
+
+```ts
+export default defineNuxtConfig({
+	// (optional) Enable the Nuxt devtools
+	devtools: { enabled: true },
+	// Enable SSG
+	ssr: false,
+	// (optional) Disable telemetry
+	telemetry: false,
+	vite: {
+		// Better support for Tauri CLI output
+		clearScreen: false,
+		// Enable environment variables
+		envPrefix: ['VITE_', 'TAURI_'],
+		build: {
+			// (optional) Support modern browsers
+			target: 'es2022',
+			// (optional) Don't minify for debug builds
+			minify: !process.env.TAURI_DEBUG ? 'esbuild' : false,
+			// (optional) Produce sourcemaps for debug builds
+			sourcemap: !!process.env.TAURI_DEBUG,
+		},
```

Comment:
No but I'd argue showing them off will help people and since the comment says it's optional I think it'd help more people than it might potentially "harm". Remove it if you wanna keep things minimal

Reply:
I actually cleaned this up a bit in CTA https://github.com/tauri-apps/create-tauri-app/commit/b586eef0c5d5462dbcc84aa1704dd41b03e0e4c2, the defaults of vite build options are fine for tauri.

Reply:
Let's remove this then to match CTA.